content/post/kpis.md

```diff
@@ -0,0 +1,91 @@
+
+<li> How KPIs should be structured within an organization
+<li> How to best partner with stakeholders in determining the best KPIs for them
+
+## KPIs should fit into the organization's framework
+
+Key Performance Indicators are indicators of progress towards some objective. They are important because they provide direction for folks using them as well as visibility for management to understand how the business is trending. 
+
+However, it's easy to overdo KPIs, e.g., just because some are good, does not mean that more is better. KPIs should be sensible across teams and up-and-down the organization.
+
+### KPIs should tell a story
+
+KPIs should tell a story -- ideally the story that your organization wants to tell. KPIs help folks transform a qualitative narrative ("We're bringing joy to under served communities") into a quantitative measure ("% of engaged users within the past 30 days"). 
+
+The story should start at the company level and filter down to every team and individual. At each layer there should exist a set of KPIs that are connected, however loosely, like a [DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph).
+
+![DAG](/img/kpi_dag.png)
+
+Figuring out KPIs are related is a useful exercise and ensures everyone is working towards the same goals. I like to sketch these out and make visible to all teams.
+
+### Ensure that it makes sense for your organization
+
+Some KPIs may be very specific indeed, and only a few folks on a certain team will care. But as indicated above, every KPI is related in some way to other KPIs elsewhere in the organization. Teams can't optimize in silos. Improving a KPI on one team should not come at the expense of another team. One common example of this is bringing in lots of leads that don't end up converting down stream. Marketing may be crushing their top line goals but Product ends up overloaded with unqualified users who drop out of the acquisition funnel.
+
+Another example: Finance wants to achieve a specific profit margin in a business line, and they identify some cost cutting measures, which mean that some teams need to change their priorities. Both of those functions need to work together to determine how they'll achieve their objectives without creating friction.
+
+## Identify the right KPIs for each team
+
+Once a KPI framework is in place, the analyst should consider how to select the best set of KPIs at each level (Org, Team, Individual). The first step is to identify someone that is ultimately responsible (accountable) for the KPI, even if multiple folks contribute to it. You should find one stakeholder per team or function that will act as your [thought partner]({{< relref "culture_of_partnership.md" >}}) and KPI owner. Work with this individual to iron out the details about:
```

Comment:
I think before this you could add a meta-section on the analyst / analytics team role in KPIs (and _why_ the analytics team should be evolved). It feels like a bit of non-sequitur dropping into this sentence about the analyst.